core/amber/src/main/scala/edu/uci/ics/texera/web/service/WorkflowService.scala

```diff
@@ -83,7 +88,8 @@ class WorkflowService(
     s"workflowId=$workflowId",
     cleanUpTimeout,
     () => {
-      ResultStorage.getOpResultStorage(workflowId).clear()
+      // TODO: ask Shengquan, what level is this doing
+//      ExecutionResourcesMapping.getResourceURIs(workflowId).clear()
```

Comment:
This is on workflow level. It removes the most recent execution state of a workflow after a specified period of inactivity.

Reply:
Got it. Since I changed the mapping to: from executionId to List[URI], each URI is pointing to the storage resource URI. The logic here would be: 
- get the latest execution id using the workflowId
- use the execution id to get the List[URI]
- for each URI, openDocument(uri).clear()
Is this correct?

Reply:
I think so, I saw every execution still overrides the previous result in line 237-241 of this file, so you only need to remove the last execution result.

Reply:
Done